docs/user_guide/build_intel_boot_bin.rst

```diff
@@ -0,0 +1,811 @@
+   $    ./adrv9371x_a10soc.sof soc_system.rbf
+
+Building the Preloader and Bootloader Image
+```````````````````````````````````````````````````````````````````````````````
+
+This flow applies starting with release 2021_R1 / Quartus Pro version 20.1.
+For older versions of the flow see previous versions of this page on wiki
+:dokuwiki:`Altera SOC Quick Start Guide <resources/tools-software/linux-software/altera_soc_images>`.
+
+In HDL project directory, create the software/bootloader folder and clone the
+``u-boot-socfpga`` image:
+
+.. shell::
+   :no-path:
+
+   $cd ~/hdl/projects/adrv9371x/a10soc ; pwd
+    ~/hdl/projects/adrv9371x/a10soc
+   $mkdir -p software/bootloader
+   $cd software/bootloader
+   $git clone https://github.com/altera-opensource/u-boot-socfpga.git
+
+Then run the qts filter and build the preloader and bootloader images:
+
+.. shell::
+   :no-path:
+
+   $cd ~/hdl/projects/adrv9371x/a10soc/software/bootloader ; pwd
+    ~/hdl/projects/adrv9371x/a10soc/software/bootloader
+   $cd u-boot-socfpga ; pwd
+    ~/hdl/projects/adrv9371x/a10soc/software/bootloader/u-boot-socfpga
+   $git checkout rel_socfpga_v2021.07_22.02.02_pr
+   $./arch/arm/mach-socfpga/qts-filter-a10.sh ../../../hps_isw_handoff/hps.xml \
+   $   arch/arm/dts/socfpga_arria10_socdk_sdmmc_handoff.h
+   $make socfpga_arria10_defconfig
+   $make
+
+Create the SPL image:
+
+.. shell::
+   :no-path:
+
+   $cd ~/hdl/projects/adrv9371x/a10soc/software/bootloader/u-boot-socfpga ; pwd
+    ~/hdl/projects/adrv9371x/a10soc/software/bootloader/u-boot-socfpga
+   $ln -s ../../../soc_system.core.rbf .
+   $ln -s ../../../soc_system.periph.rbf .
+   $sed -i 's/ghrd_10as066n2/soc_system/g' board/altera/arria10-socdk/fit_spl_fpga.its
+   $./tools/mkimage -E -f board/altera/arria10-socdk/fit_spl_fpga.its fit_spl_fpga.itb
+
+Last but not least, create the extlinux.conf linux configuration file. This
+extlinux folder shall be copied to /BOOT partition of the SD Card:
+
+.. shell::
+   :no-path:
+
+   $cd ~/hdl/projects/adrv9371x/a10soc/software/bootloader/u-boot-socfpga ; pwd
+    ~/hdl/projects/adrv9371x/a10soc/software/bootloader/u-boot-socfpga
+   $mkdir extlinux
+   $printf "\
+   $LABEL Linux Arria10 Default\n\
+   $KERNEL ../zImage\n\
+   $    FDT ../socfpga_arria10_socdk_sdmmc.dtb\n\
+   $    APPEND root=/dev/mmcblk0p2 rw rootwait earlyprintk console=ttyS0,115200n8" \
+   $    > extlinux/extlinux.conf
+
+Configuring the SD Card
+```````````````````````````````````````````````````````````````````````````````
+
+Below is the commands to create the preloader and bootloader partition using
+the Kuiper Linux image as a starting point.
+Please check every command before running, especially configuring target
+device mountpoints accordingly
+(here as ``/dev/sdz`` with partition 1 mounted at ``/media/BOOT/``).
+
+Flash the SD Card with the Kuiper Linux image.
+
+.. shell::
+
+   $time sudo dd if=./2023-12-13-ADI-Kuiper-full.img of=/dev/sdz status=progress bs=4194304
```

Comment:
You can point here to Kuiper documentation (on wiki or analog.com for now) so people can use the latest.

Reply:
Already pointed in section 'Using Kuiper Linux pre-built images'. There's a link directly to Wiki Kuiper images page